teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -158,3 +158,933 @@ With regards to this checklist, reviewers should do the following:
+
+* **Code Snippet**
+
+Include `code_snippet` like below:
+
+```text
+## code_snippet
+def pls_work(): return True... maybe
+```
+
+* **Image**
+
+Include an `image` like below:
+
+```text
+## image
+<img src="dee.jpg">
+```
+
+* **Final Example:**
+
+Note: You can always add more steps as long as you follow the above syntax.
+
+```text
+# concept_name
+Concept name
+
+# image_folder
+Topic1/Module2_test/concepts/
+
+# 1-2-1 Step 1
+
+## name
+How to install VSCode
+
+## md_content
+Hello world in js. This is how you do it
+
+## code_snippet
+def pls_work(): return True... maybe
+
+## image
+<img src="dee.jpg">
+```
+
+#### Checkpoint README Format
+
+**Filename**
+
+For the file name, name it the card that you want it to be associated with. So if you wanted to create a checkpoint for 1.md, you would name the checkpoint file `1-checkpoint.md`
+
+**Name**
+
+Every checkpoint needs a `name`:
+
+```text
+# name
+Code Check in
+```
+
+**Cards Folder**
+
+Every checkpoint needs a `cards_folder`:
+
+```text
+# cards_folder
+Topic1/Module7/Activity_13/cards/
+```
+
+**Checkpoint Type**
+
+This is to define the type of checkpoint. Put `checkpoint_type` in a markdown `h1` tag.
+
+Checkpoints could be **ONE** of the following:
+
+* Video
+* Image
+* Short Answer
+* Multiple Choice
+* Autograder
+
+```text
+# checkpoint_type
+Video
+```
+
+**Instruction**
+
+This is what you would tell the user to do for a checkpoint. Put the word `instruction` in a markdown `h1` tag. Put the instruction content under the `h1` tag like so:
+
+```text
+# instruction
+Submit a photo your Minsweeper code working
+```
+
+**Other Fields**
+
+* **Criteria \(Required for Video and Image Checkpoints Only\)**
+
+Criteria correspond to a rubric for TAs to grade a student's Image or Video Checkpoint.
+
+You must declare where the criteria for a checkpoint would begin. You would do this by putting the word `criteria` in a markdown `h1` tag like so:
+
+```text
+# criteria
+```
+
+After that you can list each criteria in a markdown `h2` tag with a number at the end like so:
+
+```text
+## criteria_1
+Does the student's board look like this:
+```
+
+* **Multiple Choice \(Only for MC Checkpoints\)**
+
+You must declare where the choices for the checkpoint would begin. You would do this by putting the words `mc_choices` in a markdown `h1` tag like so:
+
+```text
+# mc_choices
+```
+
+Next you would define each choice in a markdown `h2` tag with a number at the end like so:
+
+```text
+## choice_1
+```
+
+* **Files to send \(Autograder Checkpoint Only\)**
+
+These are the files that the student will submit if the student chooses to submit through their code to the `cli`.
+
+```text
+# files_to_send
+main.py class1.py class2.py
+```
+
+**Test file location \(Autograder Checkpoint Only\)**
+
+This field is used to tell where the test case folder is for the checkpoint.
+
+```text
+# test_file_location
+Topic1/Module7/Activity_13/Tests/test_1
+```
+
+#### Checkpoint Examples
```

Comment:
for these examples, they should each have context, so that readers know what each checkpoint will accomplish.